data/json/itemgroups/SUS/lodge.json

```diff
@@ -0,0 +1,80 @@
+[
+  {
+    "id": "SUS_hunting_archery",
+    "type": "item_group",
+    "//": "items found in an archery space",
+    "subtype": "collection",
+    "entries": [
+      { "item": "compbow", "prob": 50 },
+      { "item": "compbow_high", "prob": 20 },
+      { "item": "compbow_low", "prob": 40 },
+      { "item": "recurbow", "prob": 30 },
+      { "item": "reflexrecurvebow", "prob": 10 }
+    ]
+  },
+  {
+    "id": "SUS_hunting_rifle",
+    "type": "item_group",
+    "//": "items found in a rifle hunting space",
+    "subtype": "collection",
+    "entries": [
+      { "item": "ar15", "prob": 150, "charges-min": 0, "charges-max": 30 },
+      { "item": "marlin_9a", "prob": 20, "charges-min": 0, "charges-max": 19 },
+      { "item": "remington700_270", "prob": 10, "charges-min": 0, "charges-max": 4 },
+      { "item": "remington_700", "prob": 40, "charges-min": 0, "charges-max": 4 },
+      { "item": "ruger_1022", "prob": 70, "charges-min": 0, "charges-max": 10 },
+      { "item": "ruger_mini", "prob": 10, "charges-min": 0, "charges-max": 5 },
+      { "item": "win70", "prob": 20, "charges-min": 0, "charges-max": 3 },
+      { "item": "colt_lightning", "prob": 2, "charges-min": 0, "charges-max": 10 },
+      { "item": "henry_big_boy", "prob": 2, "charges-min": 0, "charges-max": 10 },
+      { "item": "weatherby_5", "prob": 2, "charges-min": 0, "charges-max": 3 }
+    ]
+  },
+  {
+    "id": "hunting_lodge_weapons",
+    "type": "item_group",
+    "//": "items found in a shotgun hunting space",
+    "subtype": "distribution",
+    "entries": [
+      { "group": "guns_shotgun_common", "prob": 30 },
+      { "group": "SUS_hunting_rifle", "prob": 20 },
+      { "group": "SUS_hunting_archery", "prob": 15 }
+    ]
+  },
+  {
+    "id": "cannibal_weapons",
+    "type": "item_group",
+    "//": "items found in a shotgun hunting space",
```

Comment:
The comment - not quite right for `cannibal_weapons` - seems to be copied from the `hunting_lodge_weapons` item group above.